.drone.yml

```diff
@@ -14,7 +14,7 @@ environment:
   PROD_KUBE_SERVER: https://kube-api-prod.prod.acp.homeoffice.gov.uk
   PROD_KUBE_CLUSTER_NAME: acp-prod
   IMAGE_NAME: clamav
-  CLAMAV_BUILD_NO: latest
+  CLAMAV_BUILD_NO: 0.4.7
```

Comment:
Are you sure you want to tie it to a version that's not latest? which means, rather than just running a build again and fixing vulnerability from CLAMV we'llhave to do code changes to figure out next version and fix it.

Are you sure the latest build has issues and older version does not?

Reply:
@hemantgoswami , ClamAV is based on the ACP ClamAV service. The latest tag was built a year ago, and version 0.4.7 was built 7 months ago. It appears that the ACP team ran a build and incremented the tag version to 0.4.7, as seen in this Artifactory:
https://quay.io/repository/ukhomeofficedigital/acp-clamav?tab=tags
https://quay.io/repository/ukhomeofficedigital/acp-clamav-http?tab=tags

However, the latest tag has not been updated to address vulnerabilities.